src/material/chips/chip.ts

```diff
@@ -150,6 +150,9 @@ export class MatChip
   /** ARIA label for the content of the chip. */
   @Input('aria-label') ariaLabel: string | null = null;
 
+  /** ARIA description for the content of the chip. */
+  @Input('aria-description') ariaDescription: string | null = null;
```

Comment:
why does this have to be an `@Input`? can't users just do [attr.aria-description]="..."

Reply:
Both the aria-label and the aria-description are an `@Input`. That's because the aria-label/aria-description doesn't actually go on the host element. For mat-chip-option, it goes on the `<button role="option"` element. For mat-chip-row it goes on the primary grid cell.

See changes in this PR to mat-chip-option.html and mat-chip-row.html.